pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -261,6 +261,18 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 		return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
 	}
 
+	if cephBlockPoolRadosNamespace.Spec.Mirroring.Enabled {
+		err = cephclient.EnableRBDRadosNamespaceMirroring(r.context, r.clusterInfo, cephBlockPoolRadosNamespace.Spec.BlockPoolName, getRadosNamespaceName(cephBlockPoolRadosNamespace), cephBlockPoolRadosNamespace.Spec.Mirroring.RemoteNamespace, cephBlockPoolRadosNamespace.Spec.Mirroring.Mode)
+		if err != nil {
+			return reconcile.Result{}, errors.Wrapf(err, "failed to enable rbd rados namespace mirroring")
+		}
+	} else {
+		err = cephclient.DisableRBDRadosNamespaceMirroring(r.context, r.clusterInfo, cephBlockPoolRadosNamespace.Spec.BlockPoolName, getRadosNamespaceName(cephBlockPoolRadosNamespace))
```

Comment:
will this fail if the mirroring is already disabled on the namespace or idempotent?

Reply:
It gives such outputs
```
sh-5.1$ rbd mirror pool disable replicapool/r11
rbd: mirroring is already disabled
sh-5.1$ 
sh-5.1$ 
sh-5.1$ rbd mirror pool disable replicapool/r11
rbd: mirroring is already disabled
sh-5.1$ 
sh-5.1$ 
sh-5.1$ 
sh-5.1$ rbd mirror pool enable replicapool/r11 image  --remote-namespace r13
 DBG: Found remote namespace : r13
sh-5.1$ 
sh-5.1$ 
sh-5.1$ rbd mirror pool enable replicapool/r11 image  --remote-namespace r13
 DBG: Found remote namespace : r13
rbd: mirroring is already configured for image mode
sh-5.1$ 
```